server/Dockerfile

```diff
@@ -17,7 +17,7 @@ ARG KEYCLOAK_DIST=https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloa
 
 USER root
 
-RUN yum update -y && yum install -y epel-release git && yum install -y jq openssl which && yum clean all
```

Comment:
Looks like you removed jq, it was used for custom scripts parsing the external ip when binding public interfaces in aws.

Reply:
@gmanolache please follow up on keycloak-user mailing list, we do not keep track of comments on closed pull requests.